CHANGELOG.md

```diff
@@ -9,6 +9,21 @@
 - Test Ref resources to IAM Roles have good assume role documents.  Example: Lambda Function Ref refers to an IAM Role that can be assume by Lambda.
 - More Warnings around hard coded values (Regions, AccountIds) to help with the practice of reusability
 
+### 0.5.0
+###### Features
+- Load all instances of CloudFormationLintRule in a file.  Class doesn't need to match filename.
```

Comment:
Weird bonus space there. Perhaps rewrite to `Class doesn't need to match the filename anymore`?